.github/workflows/ci.yml

```
@@ -0,0 +1,39 @@
+        uses: Swatinem/rust-cache@v2
+        with:
+          cache-on-failure: true
+          cache-all-crates: true
+          cache-workspace-crates: true
+
+      - name: Set up toolchain
+        uses: moonrepo/setup-toolchain@v0
+        with:
+          auto-install: true
+
+      - name: Run Checks 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)